SickBeard.py

```diff
@@ -20,6 +21,7 @@
 # Check needed software dependencies to nudge users to fix their setup
 
 # pylint: disable=W0703
+#                 Catching too general exception
```

Comment:
=P Several of these

Also, many of thse can be disabled in your pylintrc. I have been fine tuning mine to soon add it to the repo